packages/framework/react/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@blocksuite/react",
+  "version": "0.12.0",
+  "description": "Lit renderer for blocksuite store",
```

Comment:
"Lit renderer?" 👀

I'm not sure if this package should belong to framework, since it may be used to wrap preset components like editors and fragments. /cc @Saul-Mirone